requirements-dev.txt

```
@@ -2,7 +2,7 @@ black~=22.0
 stestr>=3.0.0
 astroid==2.9.3
 pylint==2.12.2
-qiskit-sphinx-theme>=1.6
+qiskit-sphinx-theme~=1.12.0
```

Even if you don't switch to the new theme yet, you should still upgrade.

```suggestion
qiskit-sphinx-theme~=1.13.0
```